lib/compat/experimental/block-editor-settings-mobile.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * Adds settings to the mobile block editor.
+ *
+ * @package gutenberg
+ */
+
+/**
+ * Adds settings to the mobile block editor.
+ * 
+ * This is used by the settings REST endpoint and it should land in core
+ * as soon as lib/class-wp-rest-block-editor-settings-controller.php does.
+ *
+ * @param array $settings Existing block editor settings.
+ *
+ * @return array New block editor settings.
+ */
+function gutenberg_get_block_editor_settings_mobile( $settings ) {
```

Comment:
This has been extracted to the experimental folder because we don't know yet what's going to happen with the REST Endpoint this method complements. To land in core in the same release as the endpoint.